tests/unit/compiler/port_spec/init_test.py

```diff
@@ -155,25 +154,6 @@ def test_port_spec_throws_full_address_error(self):
         with self.assertRaises(ReusedHostFullAddress):
             get_port_spec_document(expanded_spec, '192.168.5.10')
 
-    def test_port_spec_throws_container_port(self):
-        expanded_spec = {'apps':
-                                {'gcweb':
-                                         {'host_forwarding':[{'host_name': 'local.gc.com',
-                                                              'host_port': 80,
-                                                              'container_port': 80,
-                                                              'type': 'http'},
-                                                             {'host_name': 'local.gc.com',
-                                                             'host_port': 81,
-                                                              'container_port': 80,
```

Comment:
this tests if the container ports are the same.  not if the host port is the same